[2024-04-25T06:22:35.010+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-25T06:22:35.031+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_minio_s3_v02.sensor_minio_s3 scheduled__2023-06-30T00:00:00+00:00 [queued]>
[2024-04-25T06:22:35.039+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_minio_s3_v02.sensor_minio_s3 scheduled__2023-06-30T00:00:00+00:00 [queued]>
[2024-04-25T06:22:35.061+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-04-25T06:22:35.085+0000] {taskinstance.py:2327} INFO - Executing <Task(S3KeySensor): sensor_minio_s3> on 2023-06-30 00:00:00+00:00
[2024-04-25T06:22:35.091+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=12773) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-25T06:22:35.091+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_minio_s3_v02', 'sensor_minio_s3', 'scheduled__2023-06-30T00:00:00+00:00', '--job-id', '2382', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_minio_s3.py', '--cfg-path', '/tmp/tmp37ee_wrf']
[2024-04-25T06:22:35.092+0000] {standard_task_runner.py:91} INFO - Job 2382: Subtask sensor_minio_s3
[2024-04-25T06:22:35.092+0000] {standard_task_runner.py:63} INFO - Started process 12781 to run task
[2024-04-25T06:22:35.145+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_minio_s3_v02.sensor_minio_s3 scheduled__2023-06-30T00:00:00+00:00 [running]> on host a55babecb56c
[2024-04-25T06:22:35.217+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_minio_s3_v02' AIRFLOW_CTX_TASK_ID='sensor_minio_s3' AIRFLOW_CTX_EXECUTION_DATE='2023-06-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-06-30T00:00:00+00:00'
[2024-04-25T06:22:35.223+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-25T06:22:35.232+0000] {baseoperator.py:399} WARNING - S3KeySensor.execute cannot be called outside TaskInstance!
[2024-04-25T06:22:35.234+0000] {s3.py:110} INFO - Poking for key : s3://***/data.csv
[2024-04-25T06:22:35.246+0000] {base.py:84} INFO - Using connection ID 'minio_conn' for task execution.
[2024-04-25T06:22:35.247+0000] {connection_wrapper.py:387} INFO - AWS Connection (conn_id='minio_conn', conn_type='aws') credentials retrieved from extra.
[2024-04-25T06:22:35.250+0000] {logging_mixin.py:188} WARNING - <string>:9 AirflowProviderDeprecationWarning: extra['host'] is deprecated and will be removed in a future release. Please set extra['endpoint_url'] instead
[2024-04-25T06:22:35.407+0000] {base.py:294} INFO - Success criteria met. Exiting.
[2024-04-25T06:22:35.407+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-25T06:22:35.412+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=dag_with_minio_s3_v02, task_id=sensor_minio_s3, execution_date=20230630T000000, start_date=20240425T062235, end_date=20240425T062235
[2024-04-25T06:22:35.443+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-25T06:22:35.455+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-25T06:22:35.456+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
